frontend/src/app/mymeetups/open-requests/open-requests.component.html

```diff
@@ -0,0 +1,17 @@
+<h2>Meine Anfragen</h2>
```

Comment:
Ich habe jeweils das 'component' im Filenamen der html Files weggelassen. Für mich ist es so angenehmer zu lesen in der Nav.

Reply:
Neuer Name open-requests.html